docs/UserGuide.md

```diff
@@ -209,22 +209,19 @@ Examples:
 
 Schedules a meeting with the specified person.
 
-Format: `schedule INDEX s/START_DATETIME e/END_DATETIME`
+Format: `schedule INDEX s/START_DATETIME e/END_DATETIME [mr/MEETING_REMARK]`
 
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​
-* START_DATETIME must be after the system's current datetime.
-* END_DATETIME must be after START_DATETIME
+* `START_DATETIME` must be after the system's current datetime.
+* `END_DATETIME` must be after `START_DATETIME`
 * There can be **at most** one scheduled meeting with a person.
+* `MEETING_REMARK` has the same constraints as `REMARK` as stated above.
```

Comment:
I think MEETING_REMARK also have a character limit constraint? Perhaps it should be a separately listed constraint at the start of the UG?

Reply:
Added MEETING_REMARK format to the top of UG